lntemp/harness_assertion.go

```diff
@@ -820,6 +837,7 @@ func (h *HarnessTest) assertPaymentStatusWithTimeout(stream rpc.PaymentClient,
 	timeout time.Duration) *lnrpc.Payment {
 
 	var target *lnrpc.Payment
+
```

Comment:
I don't really like this extra line after a var declaration... What linter is that? Maybe we should tweak or disable that one too.

Reply:
yeah it gave me a headache...it's from `wsl` I think. Somehow I ended up getting it tho, forcing us to group `var`s together and emphasize them. Also looked into this linter and it can be configured to allow `var` followed by statements. Anyway it's disabled atm.